go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	golang.org/x/crypto v0.0.0-20191011191535-87dc89f01550
 	golang.org/x/net v0.0.0-20191021144547-ec77196f6094
 	golang.org/x/xerrors v0.0.0-20191011141410-1b5146add898 // indirect
+	google.golang.org/appengine v1.6.1 // indirect
```

Comment:
I suspect the verify error is likely related to updates that were made in master for this file.

Is there anyway we can push the appengine dependency into the test/e2e go module file? Feels a bit weird to see it here.

Reply:
lol, nm I just realized it was an indirect dep and looks like it's already present in the go.sum, ignore (other than the verify error) :joy: